src/main/steps/applicant/upload-document/upload-documents-success/content.test.ts

```diff
@@ -7,14 +7,24 @@ import { generateContent } from './content';
 
 const en = {
   section: 'How your documents will be shared',
-  //email: docsEmail,
+  status: 'Your documents have been uploaded',
+  title: 'Your documents have been uploaded',
+  uploadAgain: 'Upload another document',
   continue: 'Continue',
+  remove: 'Remove',
+  sucess: 'Success',
+  documentDetails: 'Your documents for ',
 };
 
 const cy: typeof en = {
   section: 'How your documents will be shared',
-  //email: docsEmail,
-  continue: 'Continue',
+  status: 'Your documents have been uploaded',
```

Comment:
Can you please add language tag identifier for all welsh contents please.

Reply:
Language tags added for welsh content